src/App.js

```diff
@@ -301,6 +228,21 @@ const App = () => {
   const WEEKDAYS = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday']
 
   const [filterByDay, setFilterByDay] = useState(null)
+  const [searchTerm, setSearchTerm] = useState("");
+  const [searchResults, setSearchResults] = useState([]);
+  const handleSearchTermChange = event => {
+    setSearchTerm(event.target.value);
+  };
+  useEffect(() => {
+    let results = PICKUPS.filter(pickup => {
+      // we want to return true when at least one attribute match
+      for (let property in pickup) {
+        var pickupData = pickup[property];
+        if ((typeof pickupData !== 'boolean') && (pickupData.toLowerCase().includes(searchTerm.toLowerCase()))) return true;
```

Comment:
I was hoping for something like:
```suggestion
        let attribute = pickup[property]
        let attributeIsSubsetOfSearchTerm = attribute.toLowerCase().includes(searchTerm.toLowerCase())
        if ((typeof attribute !== 'boolean') && (attributeIsSubsetOfSearchTerm)) return true;
```
But I won't block the merge on this.

Reply:
Makes sense!  I'll add that modification so the code can have better quality.